invenio_rdm_records/services/services.py

```diff
@@ -128,3 +128,26 @@ def oai_result_item(self, identity, oai_record_source):
             record,
             links_tpl=self.links_item_tpl,
         )
+
+    @unit_of_work()
+    def reindex_latest_first(self, identity, id_, extra_filter=None, uow=None):
```

Comment:
naming is open for suggestions.

question: should we have this in `drafts-resources` or leave it here? The idea is to have a way of re-indexing the latest version of a record first and siblings afterwards.